datafusion/core/src/physical_plan/streaming.rs

```diff
@@ -142,6 +148,45 @@ impl ExecutionPlan for StreamingTableExec {
         })
     }
 
+    fn fmt_as(
```

Comment:
I think this will (logically) conflict with https://github.com/apache/arrow-datafusion/pull/6835 and thus will need a merge up from main to resolve